src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My To Do List</title>
+</head>
+<body>
+  <section>
+    <div class="todo-wrap">
+      <div class="header">
+        <p class="title">Today's To Do</p>
+        <svg class="w-6 h-6 refresh" fill="#8FF3FF" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 4v5h.582m15.356 2A8.001 8.001 0 004.582 9m0 0H9m11 11v-5h-.581m0 0a8.003 8.003 0 01-15.357-2m15.357 2H15"></path></svg>
+      </div>
+      <input type="text" id="todo-input" class="todo-input" placeholder="Add to your list...">
+      <ul id="myUL">
+        
+      </ul>
+      <button id="clear-button" class="clear-button">Clear all completed</button>
+    </div>
+  </section>
```

Comment:
![Screenshot from 2022-09-29 01-00-19](https://user-images.githubusercontent.com/93039359/192878090-f4f1b3a1-e670-44bf-ab12-d8096a6888e2.png)

- [ ] Kindly make your tasks look similar to the design as suggested by the previous reviewer :thinking: You have tried it but it would be nice if your tasks should expand on the whole container by width :+1: